packages/@okta/vuepress-site/docs/guides/terraform-enable-org-access/main/index.md

```diff
@@ -26,83 +26,113 @@ Create an Okta app and credentials that Terraform uses to manage the objects in
 
-> **Note:** If you use an external tool to generate a key pair, audit the code beforehand to maintain org security.
+> **Important:** Key pair authentication is the most secure option. Use other options such as API tokens only when the tradeoff of decreased security is worth it to you.
 
-This guide uses Okta to generate the public/private key pair:
+### Use Okta to generate the public/private key pair
 
 1. In the Admin Console, open the service app and select **General**.
 1. In the **Client Credentials** section, click **Edit** to change the client authentication method.
 1. Select **Public key / Private key** as the **Client authentication** method.
 1. Click **Add key**.
 1. In the Add a public key dialog, click **Generate new key**.
-1. In the **Private key** section, click **PEM**.
+1. In the **Private key** section, click **PEM** to show the PEM format (not JSON) format of the private key.
 1. Copy the PEM private key and save it as a `.key` file.
+   > **Important:** The private key appears in this dialog only once. Losing the private key requires generating a new pair of keys.
+1. Save your new key: click **Done**, **Save**, and then **Save** again.
+1. If you created multiple keys in this application, set the status of any public key currently used by Terraform to **Active**. Set the status of the other public keys to **Inactive**.  
+1. For a production deployment, securely store the private key in a Terraform secrets management system or other key management system.
 
-   > **Note:** The private key only appears in this dialog once. Losing the private key requires generating a new pair of keys.
-1. Click **Done**, **Save**, and then **Save** again.
-
-> **Note:** Set the status of any public key currently used by Terraform to **Active**, and set the status of the other public keys to **Inactive**.
+### Use an external tool to generate the public/private key pair
 
-Check that the generated private key is in PKCS#1 format, which is the format required by the Okta Terraform Provider. In that format, the file that contains the private key begins with `-----BEGIN RSA PRIVATE KEY-----`.
-
-If the key isn't in the right format, convert it to the correct format using the OpenSSL command line program:
+1. Audit your external tool to generate a key pair for security. Follow organization best practices and audit any code beforehand to maintain org security. Never use an untrusted third-party web site to generate keys for production systems.
+1. Check that the generated private key is in PKCS#1 format, which is the format required by the Okta Terraform provider. In that format, the file that contains the private key begins with `-----BEGIN RSA PRIVATE KEY-----`. If the key isn't in the right format, convert it to the correct format using the OpenSSL command line program:
```

Comment:
This is important, but is this a step or is it a paragraph that introduces a step? I think it's the latter.

Reply:
```suggestion

Check that the generated private key is in PKCS#1 format, which is the format required by the Okta Terraform provider. In that format, the file that contains the private key begins with `-----BEGIN RSA PRIVATE KEY-----`. If the key isn't in the right format, convert it to the correct format using the OpenSSL command line program:
```